src/components/credits-footer.js

```diff
@@ -60,7 +61,7 @@ function CreditsFooter({ documentOrRevision, type }) {
 
 CreditsFooter.propTypes = {
   documentOrRevision: PropTypes.oneOfType([documentShape, documentRevisionShape]).isRequired,
-  type: PropTypes.oneOf(['document', 'revision']).isRequired
+  type: PropTypes.oneOf(Object.values(DOCUMENT_TYPE)).isRequired
```

Comment:
Could you please also replace the magic string `'document'` a few lines above with `DOCUMENT_TYPE.document`?

Reply:
:+1: